src/best_practices.md

```diff
@@ -59,8 +59,8 @@ pub unsafe extern "C" foo_create() -> *mut Foo {
 
 Likewise, you must document all assumptions. If a function consumes the 
 resource being pointed to by a pointer then you should state that. For 
-example you might try to add some `Point` object to the `Foo` object we created
-earlier.
+example you might try to add some `Point` object to the `Foo` object just
+created earlier.
```

Comment:
I feel like the "earlier" in "... just created earlier." is probably no longer necessary, seeing as "just created" means pretty much the same thing as "earlier". What are your thoughts?